sunscreen_tfhe/src/entities/glwe_ciphertext.rs

```diff
@@ -72,10 +74,7 @@ where
         assert_eq!(data.len(), GlweCiphertextRef::<S>::size(params.dim));
 
         GlweCiphertext {
-            data: data
-                .iter()
-                .map(|x| Torus::from(*x))
-                .collect::<Vec<Torus<S>>>(),
+            data: AVec::from_iter(SIMD_ALIGN, data.iter().map(|x| Torus::from(*x))),
```

Comment:
Maybe this could also be made a macro or function so that the user can use it more like a standard `from_iter` method. Which is to say `SIMD_ALIGN` would be hidden.